### DISCUSSION TITLE
Table对于嵌套属性能否生成一列？

### DISCUSSION THREAD

**Discussion by yangf85** [dotnetcore/BootstrapBlazor#3220] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/42733580/36bc55cc-7957-4f08-95b5-1c8628371903)
如果是直接属性，可以显示，嵌套的就显示不了，是用法错误还是不支持呢？

### Steps To Reproduce

[测试案例](https://github.com/yangf85/BootstrapTest)

**Comment by bb-auto** [comment]
@yangf85 Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment]
@yangf85 猜不出来。你数据库服务有问题吧，这种问题去讨论组不要上来就 BUG

**Reply by yangf85** [reply]
我搞了个测试，确实是不能显示，测试地址我也放上面了，大佬有空帮忙看看吧。
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/42733580/e7cd36ea-461b-4226-b0bd-3e3ec30a75be)
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/42733580/e0f46ede-39ca-4d12-b8c3-41f0f3cebef4)

**Reply by ArgoZhang** [reply, marked as answer]
移除 `AutoGenerateColumns` 可现实，嵌套类处理逻辑没有简单类简单，尽量用模板，也不复杂，实际开发中，我从不用这种写法

我看了一下源码，不显示的原因是这样的

`AutoGenerateColumns` 参数会自动将绑定模型的属性转化成表格列，但是如果绑定模型的属性是另外一个模型时，出去性能考虑（而且可能导致死循环）不在进行展开，所以 `AutoGenerateColumns` 仅对第一层模型有效。

`TableColumns` 模板中给定的绑定列又刚好是复杂类型的属性，组件内部也无法解析，复杂类型的类，不要使用 `AutoGenerateColumns`
下个版本最更多支持，稍后发布 `v8.4.1` 能支持的好一些

Release `8.4.1`

https://github.com/dotnetcore/BootstrapBlazor/pull/3234?wt.mc_id=DT-MVP-5004174
